source/supported-hardware/whl-cfl-crb.txt

```diff
@@ -24,17 +20,24 @@ The output images are generated under ``Outputs`` directory.
 Stitching
 ^^^^^^^^^^
 
-1. Gather |WHL| or |CFLR| IFWI firmware image
+1. Gather |WHL|, |CFLR| or |UPX| IFWI firmware image
 
-  This image contains additional firmware ingredients that are required to boot on |WHL| or |CFLR|.
+  Users can either download the full IFWI image if the IFWI image release is available, or read the exising IFWI image on the board using SPI programmer.
+  This image contains additional firmware ingredients that are required to boot on |WHL|, |CFLR| or |UPX|.
 
 .. note::
-  ``StitchLoader.py`` currently only supports stitching with boot guard feature **disabled**. 
+  ``StitchLoader.py`` currently only supports stitching with boot guard feature **disabled**.
 
 
-2. Stitch |SPN| images into downloaded BIOS image::
+2. Stitch |SPN| images into downloaded IFWI image::
 
-    python Platform/CoffeelakeBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE_NAME> -s Outputs/cfl/Stitch_Components.zip -o <SBL_IFWI_IMAGE_NAME>
+    For Whiskey Lake and Coffee Lake Refresh CRB boards:
+      python Platform/CoffeelakeBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE_NAME>
+             -s Outputs/cfl/Stitch_Components.zip -o <SBL_IFWI_IMAGE_NAME>
```

Comment:
should it be "-s Outputs\tgl\SlimBootloader.bin" instead of "-s Outputs/cfl/Stitch_Components.zip" here?

Reply:
good catch!  will fix it.